test/mailers/mailer_test.rb

```diff
@@ -37,10 +38,17 @@ def computed_from
       def computed_reply_to
         "reply_to@example.com"
       end
+
+      def computed_to
+        "to@example.com"
+      end
     end
 
     mail = TestMailerWithDefault.confirmation_instructions(create_user, "confirmation-token")
-    assert mail.from, "from@example.com"
-    assert mail.reply_to, "reply_to@example.com"
```

Comment:
https://docs.ruby-lang.org/en/2.1.0/Test/Unit/Assertions.html

These assertions will ALWAYS pass

assert mail.from - checks if mail.from is truthy (which it always is)
"from@example.com" - this second argument is completely ignored